00_Execute_Script.bat

```diff
@@ -1,9 +1,12 @@
 @echo off
 rem 00_Execute_Script.bat
 rem http://www.neko.ne.jp/~freewing/software/build_fritzing_100_windows/
-rem Copyright (c) 2023 FREE WING,Y.Sakamoto
+rem Copyright (c) 2025 FREE WING,Y.Sakamoto
 echo %0
-cd /d \00_fritzing
+rem Load configuration
+if exist directories_config.bat call directories_config.bat
```

Comment:
`if exist directories_config.bat call directories_config.bat`
cange to 
`call directories_config.bat`

The `directories_config.bat` file always exists, so there is no need to check for its existence .
Of course , this also applies to batch files 01 to 20 .